src/group/mls_group/new_from_welcome.rs

```diff
@@ -150,6 +173,7 @@ impl MlsGroup {
                 secret_tree: RefCell::new(secret_tree),
                 tree: RefCell::new(tree),
                 interim_transcript_hash,
+                add_ratchet_tree_extension: false,
```

Comment:
Because it's not. This is an implementation decision we have to take how we want the logic to be here. Currently I only set this to `true` if we enable it explicitly. There's no way to currently do this for group that a client joins. We can either add a function to do this, or enable this if the welcome had it. But then there's not really a point here to enable it as the group's established at this point and we don't care about this flag anymore.

Reply:
I guess this mandates some discussion. When joining a group from a `Welcome` message, the presence (or absence) of the Ratchet Tree Extension in the GroupInfo decides on whether that group uses that extension or not. To fully support extensions, we need to persist what extensions the group supports. In this particular case, we need to know what to do when constructing `Welcome` messages for other members we invite.
I'm fine postponing that for a later PR with a corresponding issue, I just want to point out that this is incomplete.

Reply:
Right there are two separate issues here. Because it's a little more I'll do them in a follow up (#186).
The existence of this extension should *not* influence the extensions used for the group. Instead the `extensions` field in the `GroupContext` should. But that's not implemented at all. Looks like https://github.com/mlswg/mls-protocol/pull/296 wasn't caught.